package.json

```diff
@@ -23,7 +23,7 @@
     "eslint": "^4.5.0",
     "eslint-config-airbnb": "^15.1.0",
     "eslint-plugin-import": "^2.7.0",
-    "eslint-plugin-jest": "^21.0.2",
+    "eslint-plugin-jest": "^21.2.0",
```

Comment:
Do not update the eslint dependencies individually. Because it has some peer dependencies with other libraries. If you really want to update it to the newest version, you should use the command that I added in the Makefile.  Just running the following command in terminal `make upgrade_eslint`. More details could be found in Makefile.